Cargo.toml

```diff
@@ -15,47 +18,31 @@ repository = "https://github.com/amethyst/amethyst"
 readme = "README.md"
 license = "MIT/Apache-2.0"
 
+# FIXME: remove after all existing examples working
+autoexamples = false
+
 [features]
-# TODO: include animation, ui
```

Comment:
`animation` and `ui` were removed from the default features for the legion port. The comment is a reminder to re-add them when enough porting is done.